requirements.txt

```diff
@@ -3,3 +3,4 @@ conan==1.60.2
 
 pre-commit==2.20.0
 cpplint==1.6.1
+pytest==7.4.1
```

Comment:
do we really need this in the main requirements.txt?

Reply:
Moved to test requirements.